apps/dotcom/sync-worker/src/TLPostgresReplicator.ts

```diff
@@ -335,6 +340,7 @@ export class TLPostgresReplicator extends DurableObject<Environment> {
 
 	private handleEvent(row: postgres.Row | null, event: postgres.ReplicationEvent) {
 		this.lastPostgresMessageTime = Date.now()
+		getStatsDurableObjct(this.env).recordReplicatorPostgresUpdate()
```

Comment:
could we throttle this? just worried about adding O(n) extra IO operations to the replicator, might make deploys even slower until we fix the reboot issue.